website/content/en/v0.25.0/getting-started/migrating-from-cas/_index.md

```diff
@@ -90,7 +90,7 @@ One for your Karpenter node role and one for your existing node group.
 
 First set the Karpenter release you want to deploy.
 ```bash
-export KARPENTER_VERSION=v0.25.0
+export KARPENTER_VERSION={{< param "latest_release_version" >}}
```

Comment:
Watch out for this kind of mismatch:
https://github.com/aws/karpenter/blob/v0.24.0/website/config.yaml#L69
(0.23.0)

Reply:
good call, thanks!